ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Extended.hs

```diff
@@ -194,6 +197,19 @@ encodeExtLedgerState encodeLedgerState
                            encodeChainDepState
                            encodeAnnTip
 
+encodeExtLedgerState' ::
```

Comment:
Improve the name? The postfix tick named seemed fine when it was a local definition, but now it's an exported top-level definition and the distinction from `encodeExtLedgerState` seems non-obvious to me.

Maybe something like `encodeExtLedgerStateViaEncodeDisk` or `encodeDiskExtLedgerState`

Reply:
Renamed to `encodeDiskExtLedgerState`, same for decoding